### ISSUE TITLE
Implement functionality to flash ESP via webinterface

### ISSUE THREAD

**Issue by tpanajott** [NSPManager/NSPanelManager#49]
Using the ESP web tools (https://esphome.github.io/esp-web-tools/) it is possible to flash the ESP from the web interface. This does however require HTTPS to be used.

**Comment by cablesandcoffee** [comment]
This can be closed right?

**Comment by tpanajott** [comment]
Yes, I agree. This has now been implemented on nspanelmanager.com. Yet again, thanks to @sebastienvermeille !
